### ISSUE TITLE
[2026-02-11] Daily Papers (56 papers)

### ISSUE THREAD

**Issue by Shininglay** [Shininglay/get-daily-noti#14]
# 📅 Daily Paper Update: 2026-02-11

**今日发现 56 篇近期(30天内)相关论文**

---

### [Ultrafast scintillating metal-organic framework films](https://www.nature.com/articles/s41467-026-68546-6)
- **关键词**: `GaN`
- **发布时间**: 2026-02-10

### [Spectral Fingerprinting of Engineered Nanomaterials for Precision Biosensing](https://pubs.acs.org/doi/10.1021/acsnano.5c18348?af=R)
- **关键词**: `sensing`
- **发布时间**: 2026-01-26T08:00:00Z

### [Engineering Adaptive Hydrogen Bond Networks in Metal–Organic Frameworks for Bioinspired H2O2 Catalysis Enhancement](https://pubs.acs.org/doi/10.1021/acsnano.5c16726?af=R)
- **关键词**: `GaN`
- **发布时间**: 2026-01-31T08:00:00Z

### [Organic Thin-Film Transistors on Ultrathin Elastomeric Substrates for Conformable, Stretchable, and Breathable Electronics](https://pubs.acs.org/doi/10.1021/acsnano.5c17186?af=R)
- **关键词**: `stretchable`, `GaN`
- **发布时间**: 2026-01-30T08:00:00Z

### [Temperature-Responsive Xylene Isomer Separation via Biomimetic Window Confinement and Defect-Engineered Nanospace in Metal–Organic Frameworks](https://pubs.acs.org/doi/10.1021/acsnano.5c18835?af=R)
- **关键词**: `GaN`
- **发布时间**: 2026-01-26T08:00:00Z

### [In-Sensor Computing Tactile Devices for Well-Defined Confined Space Interactions](https://pubs.acs.org/doi/10.1021/acsnano.5c19078?af=R)
- **关键词**: `sensor`
- **发布时间**: 2026-01-31T08:00:00Z

### [Pattern-Verifiable Heterowalkers Scaffolded on a DNA Origami Nanosheet for Multiplexed Profiling of miRNAs](https://pubs.acs.org/doi/10.1021/acsnano.5c20466?af=R)
- **关键词**: `nanosheet`
- **发布时间**: 2026-01-26T08:00:00Z

### [Nanocrystal-to-Ligand Interfacial Thermal Transport: Plasmonic Indium Tin Oxide to Perylene Diimide Adsorbates](https://pubs.acs.org/doi/10.1021/acsnano.5c22120?af=R)
- **关键词**: `GaN`
- **发布时间**: 2026-01-28T08:00:00Z

### [Physics Guided Exponential Model Design of High Ge Content SiGe Selective Epitaxy for Gate All Around Source/Drain Applications](https://arxiv.org/abs/2602.07925)
- **关键词**: `CMOS`
- **发布时间**: Tue, 10 Feb 2026 00:00:00 -0500

### [Infrared photonics for healthcare: A roadmap for proactive and predictive health management](https://arxiv.org/abs/2602.08248)
- **关键词**: `sensor`, `sensing`
- **发布时间**: Tue, 10 Feb 2026 00:00:00 -0500

### [Estimation of Fish Catch Using Sentinel-2, 3 and XGBoost-Kernel-Based Kernel Ridge Regression](https://arxiv.org/abs/2602.08511)
- **关键词**: `sensor`
- **发布时间**: Tue, 10 Feb 2026 00:00:00 -0500

### [Split-Post Microwave Displacement Transducer with Quadratic Readout](https://arxiv.org/abs/2602.07305)
- **关键词**: `sensor`
- **发布时间**: Tue, 10 Feb 2026 00:00:00 -0500

### [Influence of Elastic Oscillations on Nucleation in Metals](https://arxiv.org/abs/2602.07826)
- **关键词**: `GaN`
- **发布时间**: Tue, 10 Feb 2026 00:00:00 -0500

### [Enhanced Sensitivity to Blackbody Radiation in Spintronic Poisson Bolometers](https://arxiv.org/abs/2512.12490)
- **关键词**: `sensing`
- **发布时间**: Tue, 10 Feb 2026 00:00:00 -0500

### [Hybrid Deep Learning Framework for CSI-Based Activity Recognition in Bandwidth-Constrained Wi-Fi Sensing](https://arxiv.org/abs/2602.06983)
- **关键词**: `sensing`
- **发布时间**: Tue, 10 Feb 2026 00:00:00 -0500

### [Adaptive Temporal Dynamics for Personalized Emotion Recognition: A Liquid Neural Network Approach](https://arxiv.org/abs/2602.06997)
- **关键词**: `GaN`
- **发布时间**: Tue, 10 Feb 2026 00:00:00 -0500

### [Guidestar-Free Adaptive Optics with Asymmetric Apertures](https://arxiv.org/abs/2602.07029)
- **关键词**: `sensor`, `sensing`
- **发布时间**: Tue, 10 Feb 2026 00:00:00 -0500

### [MTS-CSNet: Multiscale Tensor Factorization for Deep Compressive Sensing on RGB Images](https://arxiv.org/abs/2602.07056)
- **关键词**: `sensing`
- **发布时间**: Tue, 10 Feb 2026 00:00:00 -0500

### [MRI Cross-Modal Synthesis: A Comparative Study of Generative Models for T1-to-T2 Reconstruction](https://arxiv.org/abs/2602.07068)
- **关键词**: `GaN`
- **发布时间**: Tue, 10 Feb 2026 00:00:00 -0500

### [Behavior Score Prediction in Resting-State Functional MRI by Deep State Space Modeling](https://arxiv.org/abs/2602.07131)
- **关键词**: `GaN`
- **发布时间**: Tue, 10 Feb 2026 00:00:00 -0500

### [DEMIX: Dual-Encoder Latent Masking Framework for Mixed Noise Reduction in Ultrasound Imaging](https://arxiv.org/abs/2602.07163)
- **关键词**: `sensor`
- **发布时间**: Tue, 10 Feb 2026 00:00:00 -0500

### [Distributed Omniscient Observers for Multi-Agent Systems](https://arxiv.org/abs/2602.07300)
- **关键词**: `GaN`
- **发布时间**: Tue, 10 Feb 2026 00:00:00 -0500

### [Wireless Context Engineering for Efficient Mobile Agentic AI and Edge General Intelligence](https://arxiv.org/abs/2602.07321)
- **关键词**: `sensing`
- **发布时间**: Tue, 10 Feb 2026 00:00:00 -0500

### [Message Passing based Parameter Estimation in Cooperative MIMO-OFDM ISAC Systems](https://arxiv.org/abs/2602.07365)
- **关键词**: `sensing`
- **发布时间**: Tue, 10 Feb 2026 00:00:00 -0500

### [Wavelet-Domain Masked Image Modeling for Color-Consistent HDR Video Reconstruction](https://arxiv.org/abs/2602.07393)
- **关键词**: `GaN`
- **发布时间**: Tue, 10 Feb 2026 00:00:00 -0500

### [Optimal Low-Dimensional Structures of ISAC Beamforming: Theory and Efficient Algorithms](https://arxiv.org/abs/2602.07502)
- **关键词**: `sensing`
- **发布时间**: Tue, 10 Feb 2026 00:00:00 -0500

### [Beyond $\lambda/2$: Can Arbitrary EMVS Arrays Achieve Unambiguous NLOS Localization?](https://arxiv.org/abs/2602.07515)
- **关键词**: `sensor`
- **发布时间**: Tue, 10 Feb 2026 00:00:00 -0500

### [Fractional Filtering and Anomaly-Guided Diagnostics: The Local Damage Mode Extractor (LDME) for Early Gear Fault Detection](https://arxiv.org/abs/2602.07527)
- **关键词**: `GaN`
- **发布时间**: Tue, 10 Feb 2026 00:00:00 -0500

### [A Scalable Cloud-Edge Collaborative CKM Construction Framework Enabled by a Foundation Prior Model](https://arxiv.org/abs/2602.07586)
- **关键词**: `sensing`
- **发布时间**: Tue, 10 Feb 2026 00:00:00 -0500

### [MI-ISAC: Magneto-Inductive Integrated Sensing and Communication in the Reactive Near-Field for RF-Denied Environments](https://arxiv.org/abs/2602.07714)
- **关键词**: `sensing`
- **发布时间**: Tue, 10 Feb 2026 00:00:00 -0500

### [AFDM: Evolving OFDM Towards 6G+](https://arxiv.org/abs/2602.08163)
- **关键词**: `sensing`
- **发布时间**: Tue, 10 Feb 2026 00:00:00 -0500

### [An Experimental Study on Fine-Grained Bistatic Sensing of UAV Trajectory via Cellular Downlink Signals](https://arxiv.org/abs/2602.08203)
- **关键词**: `sensing`
- **发布时间**: Tue, 10 Feb 2026 00:00:00 -0500

### [Riemannian Manifold Optimization for Advanced Wireless Communications: Fundamentals and Applications](https://arxiv.org/abs/2602.08225)
- **关键词**: `sensing`
- **发布时间**: Tue, 10 Feb 2026 00:00:00 -0500

### [Beam Alignment in Multipath Environments for Integrated Sensing and Communication using Bandit Learning](https://arxiv.org/abs/2602.08380)
- **关键词**: `sensing`
- **发布时间**: Tue, 10 Feb 2026 00:00:00 -0500

### [IEEE 802.11ad-Aided 5-D Sensing with a UAV Swarm in Urban Environment](https://arxiv.org/abs/2602.08396)
- **关键词**: `sensing`
- **发布时间**: Tue, 10 Feb 2026 00:00:00 -0500

### [Reconfigurable Low-Complexity Architecture for High Resolution Doppler Velocity Estimation in Integrated Sensing and Communication System](https://arxiv.org/abs/2602.08415)
- **关键词**: `sensing`
- **发布时间**: Tue, 10 Feb 2026 00:00:00 -0500

### [A Multi-physics Simulation Framework for High-power Microwave Counter-unmanned Aerial System Design and Performance Evaluation](https://arxiv.org/abs/2602.08477)
- **关键词**: `CMOS`
- **发布时间**: Tue, 10 Feb 2026 00:00:00 -0500

### [Radar Operating Metrics and Network Throughput for Integrated Sensing and Communications in Millimeter-wave Urban Environments](https://arxiv.org/abs/2602.08495)
- **关键词**: `sensing`
- **发布时间**: Tue, 10 Feb 2026 00:00:00 -0500

### [RFSoC-Based Integrated Navigation and Sensing Using NavIC](https://arxiv.org/abs/2602.08596)
- **关键词**: `sensing`
- **发布时间**: Tue, 10 Feb 2026 00:00:00 -0500

### [Ziv-Zakai Bound for Near-Field Localization and Sensing](https://arxiv.org/abs/2602.08609)
- **关键词**: `sensing`
- **发布时间**: Tue, 10 Feb 2026 00:00:00 -0500

### [Automating the Wildfire Detection and Scheduling Pipeline with Maneuverable Earth Observation Satellites](https://arxiv.org/abs/2602.08924)
- **关键词**: `sensor`
- **发布时间**: Tue, 10 Feb 2026 00:00:00 -0500

### [Toward Accurate and Accessible Markerless Neuronavigation](https://arxiv.org/abs/2602.07052)
- **关键词**: `sensor`, `sensing`
- **发布时间**: Tue, 10 Feb 2026 00:00:00 -0500

### [Why Look at It at All?: Vision-Free Multifingered Blind Grasping Using Uniaxial Fingertip Force Sensing](https://arxiv.org/abs/2602.07326)
- **关键词**: `sensor`, `sensing`
- **发布时间**: Tue, 10 Feb 2026 00:00:00 -0500

### [Perspective-aware fusion of incomplete depth maps and surface normals for accurate 3D reconstruction](https://arxiv.org/abs/2602.07444)
- **关键词**: `sensor`
- **发布时间**: Tue, 10 Feb 2026 00:00:00 -0500

### [Information Theoretic Modeling of Interspecies Molecular Communication](https://arxiv.org/abs/2602.07474)
- **关键词**: `GaN`
- **发布时间**: Tue, 10 Feb 2026 00:00:00 -0500

### [ODELoRA: Training Low-Rank Adaptation by Solving Ordinary Differential Equations](https://arxiv.org/abs/2602.07479)
- **关键词**: `sensing`
- **发布时间**: Tue, 10 Feb 2026 00:00:00 -0500

### [Picasso: Holistic Scene Reconstruction with Physics-Constrained Sampling](https://arxiv.org/abs/2602.08058)
- **关键词**: `sensor`
- **发布时间**: Tue, 10 Feb 2026 00:00:00 -0500

### [Optimal Transmit Beamforming for MIMO ISAC with Unknown Target and User Locations](https://arxiv.org/abs/2602.08255)
- **关键词**: `sensing`
- **发布时间**: Tue, 10 Feb 2026 00:00:00 -0500

### [Controlled Flight of an Insect-Scale Flapping-Wing Robot via Integrated Onboard Sensing and Computation](https://arxiv.org/abs/2602.08328)
- **关键词**: `sensor`, `sensing`
- **发布时间**: Tue, 10 Feb 2026 00:00:00 -0500

### [A General Nonlinear Observer Design for Inertial Navigation Systems with Almost Global Stability Guarantees](https://arxiv.org/abs/2410.03846)
- **关键词**: `sensing`
- **发布时间**: Tue, 10 Feb 2026 00:00:00 -0500

### [Sensor-Noise Mitigation in Extremum Seeking Control Using Adaptive Numerical Differentiation](https://arxiv.org/abs/2501.04275)
- **关键词**: `sensor`
- **发布时间**: Tue, 10 Feb 2026 00:00:00 -0500

### [CFARNet: Learning-Based High-Resolution Multi-Target Detection for Rainbow Beam Radar](https://arxiv.org/abs/2505.10150)
- **关键词**: `sensing`
- **发布时间**: Tue, 10 Feb 2026 00:00:00 -0500

### [MARBLE-Net: Learning to Localize in Multipath Environment with Adaptive Rainbow Beams](https://arxiv.org/abs/2511.06971)
- **关键词**: `sensing`
- **发布时间**: Tue, 10 Feb 2026 00:00:00 -0500

### [Boundary adaptive observer design for semilinear hyperbolic rolling contact ODE-PDE systems with uncertain friction](https://arxiv.org/abs/2601.09223)
- **关键词**: `sensing`
- **发布时间**: Tue, 10 Feb 2026 00:00:00 -0500

### [CoBEVMoE: Heterogeneity-aware Feature Fusion with Dynamic Mixture-of-Experts for Collaborative Perception](https://arxiv.org/abs/2509.17107)
- **关键词**: `sensing`
- **发布时间**: Tue, 10 Feb 2026 00:00:00 -0500

### [Cryo-CMOS Antenna for Wireless Communications within a Quantum Computer Cryostat](https://arxiv.org/abs/2510.13627)
- **关键词**: `CMOS`
- **发布时间**: Tue, 10 Feb 2026 00:00:00 -0500
